### DISCUSSION TITLE
Expression suggestions for "repeat" event type

### DISCUSSION THREAD

**Discussion by Elairyx7301** [4ian/GDevelop#3628] [Enhancement ideas (and their technical discussions)]
## Description
The "Repeat" event type in the scene editor supports dynamic values / expressions but suggestions are not shown, which can be understood that it's not supported even though it actually is and can cause typos.

## Solution suggested
Have the "Repeat" event type support expressions suggestions.

## Related screenshots
![image](https://user-images.githubusercontent.com/83909121/154096036-33bf496c-c751-4d74-a2a4-7c88c4d8c3c8.png)

![image](https://user-images.githubusercontent.com/83909121/154096087-dc54eb71-c00f-4178-aa74-745a2170b71a.png)

![image](https://user-images.githubusercontent.com/83909121/154096210-89771e5e-83a5-4973-8a1f-c712c2d0a98f.png)

**Comment by tristanbob** [comment]
This is a great idea.  Since expressions are allowed in the quantity field of a "Repeat" event, that field should have access to auto-complete just like the other parts of the engine.

**Comment by Elairyx7301** [comment]
Am I not understanding something about this? I think turning this into a discussion is unnecessary. I can't see a reason why this change or feature is bad or can't be implemented and also can I not think of any contra aspects for this suggestion.

**Reply by D8H** [reply]
I guess that issues will only be used for bug reports and discussions for change requests.

**Reply by 4ian** [reply]
This is understandable though that this can be considered almost as a bug - it's at least a severe lack in the UI.

**Reply by Elairyx7301** [reply]
D8H, then I don't see the reason for the "Feature request" issue type all too much

**Comment by Elairyx7301** [comment]
What is the current status on this? Is this being worked on or is there a solid reason for why this should not be implemented?

**Comment by Elairyx7301** [comment]
Since I got no reply as to why this issue was converted into a discussion or what is not optimal about this feature request, I will transform this back into an issue since discussions seemingly tend get a lower priority than issues and pull requests.
I am open to criticism of my actions or the feature request itself and if you tell me then I'm convinced we can get to a result that everyone is fine with :)

**Reply by 4ian** [reply]
I'm ok with this being ported back into an issue as it's indeed a sub optimal UI/almost a bug.
